addons/point_of_sale/static/src/app/screens/receipt_screen/receipt_screen.js

```diff
@@ -32,6 +32,15 @@ export class ReceiptScreen extends Component {
         onMounted(() => {
             const order = this.pos.get_order();
             this.pos.sendOrderInPreparation(order);
+            // When the order is paid, if there is still a part of the order
+            // to send in preparation it is automatically sent
+            if (this.pos.orderPreparationCategories.size) {
+                try {
+                    this.pos.sendOrderInPreparationUpdateLastChange(order);
+                } catch (error) {
+                    Promise.reject(error);
+                }
+            }
```

Comment:
@davidmonnom I copy this behavior from 17.2 but it makes the runbot tests failing. Would you have an idea how to solve it differently ? I'm a bit puzzled by the refactoring
The point of this is to automatically order (send to kitchen printers) if an order is validated (which is currently not the case in master)